python-susecloudimagenameparser.spec

```diff
@@ -0,0 +1,47 @@
+#
+# spec file for package susecloudimagenameparser 
+#
+# Copyright (c) 2022 SUSE LLC
+#
+
+# Please submit bugfixes or comments via https://bugs.opensuse.org/
+#
+
+%{?!python_module:%define python_module() python-%{**} python3-%{**}}
+%if 0%{?suse_version} >= 1500
+%define skip_python2 1
+%endif
```

Comment:
This does not match our definitions in `ci-workflow.vml` where it is declared that Python 2.x is not supported. However if one would set up a build target for SLE 12, according to this definition the code would get built against Python 2.7 which is the default interpreter in SLE 12.

We should name the package python3-susecloudimagenameparser and set the spec file up to use only ```python3``` making the ```%if``` condition obsolete